vignettes/GenomicFiles.Rmd

```diff
@@ -0,0 +1,712 @@
+
+---
+title: "Introduction to GenomicFiles"
+author:
+  - Valerie Obenchain
+  - Michael Love
+  - Lori Shepherd
+  - Martin Morgan
+contributor:
+  - name: Sonali Kumari
+    affiliation: "Vignette translation from Sweave to R Markdown / HTML"
+date: "`r format(Sys.time(), '%B %d, %Y')`"
```

Comment:
I updated my note about this in my original comment since the original pdf has a last modified and a compiled date.

Reply:
changed